www/video/videotest.js

```diff
@@ -21,6 +23,16 @@ function AddUrl()
 
 function RemoveUrl(id) 
 {
+    urlCount -= 1;
     $(id).remove();
     return false;
 }
+function showRemoveLink(){
```

Comment:
With styling of braces, try to follow the lead with the rest of the file (even though it may be different in other js files).